examples/next/pages/ui/components/storage/file-uploader/aws-exports.js

```diff
@@ -0,0 +1,2 @@
+import awsExports from '@environments/auth/auth-with-email/src/aws-exports';
```

Comment:
This is pointing to `auth-with-email`. Later when e2e storage environment is setup, I'll switch it to that.